templates/phaser/packages/contracts/package.json

```diff
@@ -4,20 +4,18 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "build": "forge clean && forge build",
-    "deploy": "pnpm run initialize && mud deploy",
-    "deploy:hackathon": "pnpm run initialize && mud deploy --profile=hackathon-testnet",
-    "deploy:testnet": "pnpm run initialize && mud deploy --profile=lattice-testnet",
+    "build": "pnpm run build:mud && pnpm run build:abi && pnpm run build:types",
+    "build:abi": "forge clean && forge build",
+    "build:mud": "mud tablegen && mud worldgen && mud tsgen --configPath mud.config.ts --out ../client/src/mud",
+    "build:types": "rimraf types && typechain --target=ethers-v5 out/IWorld.sol/IWorld.json",
+    "deploy": "pnpm run build && mud deploy",
+    "deploy:hackathon": "pnpm run build && mud deploy --profile=hackathon-testnet",
```

Comment:
I think we can prob get rid of this now?

Reply:
also removed the profiles from foundry.toml